main/guides/ui-tutorial/starting/index.md

```diff
@@ -0,0 +1,68 @@
+# Starting Out
+
+There are a few libraries that make building Agoric UIs more convenient, all included in the [ui-kit](https://github.com/Agoric/ui-kit/tree/main) repo:
+
+- [@agoric/rpc](https://github.com/Agoric/ui-kit/tree/main/packages/rpc): Used for querying blockchain and smart contract state, and subscribing to updates.
+- [@agoric/web-components](https://github.com/Agoric/ui-kit/tree/main/packages/web-components): Used for accessing the smart wallet and making offers, as well as utilities for displaying asset balances and more.
+- [@agoric/react-components](https://github.com/Agoric/ui-kit/tree/main/packages/react-components): React components for doing all of the above and more, with convenient hooks and context for ease of use.
+
+This tutorial will make use of these libraries and provide pointers along the way for learning more about them.
+
+## Scaffolding
+
+This tutorial will make use of [@agoric/react-components](https://github.com/Agoric/ui-kit/tree/main/packages/react-components) for greatest convenience, and hence the first step will be to scaffold a new React app with [Vite](https://vitejs.dev/). Create a new workspace and use the following command to scaffold a new `react-ts` app:
+
+```
+yarn create vite my-agoric-ui --template react-ts
+```
+
+Follow the instructions outputted by the command and you should have a local dev server running with a bare-bones React app.
+
+## Hardening
+
+The next step is to install [Endo](https://github.com/endojs/endo), because the Agoric libraries depend on Hardened JavaScript. To get it set up, we'll add a few dependencies:
+
+```
+yarn add -D ses @endo/eventual-send
+```
+
+Also, we'll need to add `buffer` to use [cosmos-kit](https://github.com/cosmology-tech/cosmos-kit) later on, so let's do that now:
+
+```
+yarn add -D buffer
+```
+
+Now, create a new file `src/installSesLockdown.ts`:
```

Comment:
we could probably link to more documentation on what this file is doing. Even if we just link to docs on lockdown. As now its just "make a new file, and copy/paste this". There an opportunity to explain lockdown briefly here. Although it isnt a blocker for the dev. More informational

Reply:
Tbh I'm just having a hard time succinctly explaining it. Thinking out loud, it prevents the modification of certain intrinsics in the JS environment, and adds support for `harden` and `assert` etc. But that might lead to more questions than answers, as those things aren't really concerns for most UI devs (unless you're using https://github.com/LavaMoat/LavaMoat). It's really just needed so the agoric libraries work, because they're also used in smart contracts where those things actually matter.

We've talking about having a way to use the agoric libraries without hardening, but there hasn't been any movement on that yet. Maybe just linking to endo here is enough for now.